integration/tests/cook/test_pools.py

```diff
@@ -45,6 +44,8 @@ def test_pool_scheduling(self):
                     quota_multiplier = 1 if pool_name == default_pool else 2
                     util.set_limit(self.cook_url, 'quota', user.name, cpus=cpus * quota_multiplier, pool=pool_name)
 
+            logging.info(f"User to access: {user}")
```

Comment:
Can you make the log message more specific? Something like `test_pool_scheduling user: {user}`

Reply:
Thats a stale debugging line. :( 
Gone